dashboard/app/views/levels/_admin.html.haml

```diff
@@ -129,10 +129,14 @@
                   %tt= link_to channel[0,7], "/projects/#{params[:key]}/#{channel}"
 
       #feature_project{'style' => ('display: none;' unless !exists || !currently_featured)}
+        %div
+          special topic:
+          %input{id: "special-topic", value: project.topic}
```

Comment:
how many of these topics do we think that we'll have? if there are not too many, it might help avoid typos to have a dropdown here instead.

if we keep the text input, what will the experience be for the project validator who marks a project with a topic that has a typo in the topic name?

Reply:
Great question. We're not sure how many we'll have - I'd guess in the 10-30 ballpark over time.  We don't know the list of topics right now since the idea is new, but suspect we'll switch to a dropdown whenever they're decided on. The topic shows up in the featured projects UI table, so if a project isn't showing up in the special topics section of the gallery as expected at least there will be clues to find the mismatch between the topic and the typo.

Reply:
Sounds good, thanks for talking through it!